### ISSUE TITLE
Any chance to generate waystones from the Fabric Waystones mod?

### ISSUE THREAD

**Issue by viktordimov** [celsiusqc/totw_modded_mod#15]

**Comment by HaXrDEV** [comment]
Yeah just noticed that this is missing as well. Would be nice as i prefer the Fabric Waystones mod.

**Comment by zachsai** [comment]
I'd have to create specific version of fabric waystones java files and change all the top tower structure .nbt files to use the Fabric Waystones, which I do not want to do. I rather support the original creator of the mod instead of a fork, such as the fabric version.

You are welcome to change the .nbt file through datapack structure for yourself though. I just do not want to spend time on it at the moment.

**Comment by viktordimov** [comment]
You do in fact support the Fabric fork of the Waystones mod by BlayTheNinth.
I was asking about the "Fabric Waystones" mod or "FWaystones" by LordDeatHunter - completely unrelated except for the same idea of introducing waystones into the game.

I understand. Thanks for your reply.

**Comment by zachsai** [comment]
Yes, I mean't for the FWaystones by Lord, I will not be supporting that waystone mod unfortunately

**Comment by HaXrDEV** [comment]
For anyone interested, i created a datapack yesterday to accomplish this https://github.com/HaXrBOT/TOTW-Modded-FWaystones-Compat

**Comment by zachsai** [comment]
You can create a pull request and I'll upload the jar version to curse and modrinth @HaXrBOT

**Comment by SettingDust** [comment]
Below are 2 advice
Both of them need https://modrinth.com/mod/lithostitched
1.  Create a template pool for waystones. Inject the element with https://github.com/Apollounknowndev/lithostitched/wiki/Worldgen-Modifiers#add_template_pool_elements with condition https://github.com/Apollounknowndev/lithostitched/wiki/Worldgen-Modifiers#modifier-predicates. Modify the structure to add a jigsaw block
2. Use https://github.com/Apollounknowndev/lithostitched/wiki/Structure-Utilities#block_swap with condition. Maybe there is no need to edit the structure file
